src/repositories/user_repository.py

```diff
@@ -0,0 +1,32 @@
+from models.user import User
+from database import db
+from flask_login import LoginManager
+from flask_login import login_user, logout_user
+
+login = LoginManager()
+
+class UserRepository:
+    def __init__(self):
+        pass
+
+    def login(self, username, password):
+        user = User.query.filter_by(username = username).first()
+        if user is not None and user.check_password(password):
+            login_user(user)
```

Comment:
I think the repository should not handle login sessions, rather leave this to the service.